<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateQuestion</class>
 <widget class="QWidget" name="CreateQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>468</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>CreateQuestion</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tạo câu hỏi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nội dung câu hỏi</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDifficulty">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Độ khó</string>
   </property>
  </widget>
  <widget class="QComboBox" name="difficultyComboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Dễ</string>
    </property>
    <property name="data">
     <number>1</number>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Trung bình</string>
    </property>
    <property name="data">
     <number>2</number>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Khó</string>
    </property>
    <property name="data">
     <number>3</number>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>691</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Các đáp án</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addAnswerButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Thêm đáp án</string>
   </property>
  </widget>
  <widget class="QWidget" name="answerContainer" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="addAnswerButton_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  #6600FF ; color:white;</string>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
